api/tests/unit/domain/usecases/start-writing-campaign-assessment-results-to-stream_test.js

```diff
@@ -128,7 +131,9 @@ describe('Unit | Domain | Use Cases | start-writing-campaign-assessment-results
     const csv = await csvPromise;
 
     // then
-    expect(csv).to.equal(csvExpected);
+    const expectedFilename = `Resultats-name-${campaign.id}-${dayjs.utc().format('YYYY-MM-DD-hhmm')}.csv`;
```

Comment:
**[suggestion] :** Ne pas utiliser la lib dans les tests mais la valeur attendue.
On a une vérification mais on sait pas si ça marche du coup.

A mon avis on peut avoir deux approche plus intéressantes :
- A : On freeze la date et l'heure et on vérifie qu'on a le bon nom de fichier.
- B : On injecte le nom du fichier en paramètre du use case et on vérifie que le nom du fichier et bien ce qui est passé en paramètre. 

Perso je préfère B mais Pix fait plutôt du A je crois.

Reply:
:laughing: Effectivement c'est Dev in progress, mais tes retours sont top.
Je vais prendre l'option B

Reply:
Je prends l'option A pour finir, car le nom du fichier fait partie du métier, non ?